libs/commons/Logs_.mli

```diff
@@ -1,24 +1,30 @@
 (* Small helpers around the Logs library.
 
-   Here are the usage conventions for the Logs library level:
+   Here are the usage conventions for the Logs library level,
+   augmented from https://erratique.ch/software/logs/doc/Logs/index.html#usage
+
 
       - App, this level can be used for the standard output or console
         of an application. It should never be used by libraries.
+        Semgrep: We don't use it.
```

Comment:
we do actually; git grep Logs.app return many matches

Reply:
got it.